TrainingRNExpress/src/app/screens/BookDetails/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Text, View } from 'react-native';
+
+import { BOOKS } from '../../../constants';
+import { FIELDS } from './strings';
+
+import styles from './styles.js';
+
+class BookDetails extends Component {
+  state = { book: {}}
+
+  componentDidMount() {
+    const id = this.props.navigation.state.params.id;
```

Comment:
Does `this.props.navigation.getParam('id')` work ?
Reference: https://reactnavigation.org/docs/en/params.html